pkg/daemon/ceph/client/status.go

```diff
@@ -164,7 +166,11 @@ func StatusWithUser(context *clusterd.Context, clusterInfo *ClusterInfo) (CephSt
 
 	buf, err := context.Executor.ExecuteCommandWithOutput(command, args...)
 	if err != nil {
-		return CephStatus{}, errors.Wrapf(err, "failed to get status. %s", string(buf))
+		errMessage := "failed to get ceph status"
+		if buf != "" {
+			errMessage = fmt.Sprintf("%s. %s", errMessage, string(buf))
```

Comment:
if `buf != ""`, the errMessage will be included twice. What about returning the error here so it doesn't also get wrapped on the next line?

Reply:
Wait, should we just log the `buf` if not empty? It doesn't seem necessary for the error being returned.

Reply:
Both have been useful so far.